public/components/datasources/components/__tests__/manage_datasource_description.test.tsx

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Not sure how this got pulled in, should be on main already, can you double check/merge